Kiosk.xcodeproj/project.pbxproj

```diff
@@ -1174,7 +1174,6 @@
 				"${BUILT_PRODUCTS_DIR}/DZNWebViewController/DZNWebViewController.framework",
 				"${BUILT_PRODUCTS_DIR}/ECPhoneNumberFormatter/ECPhoneNumberFormatter.framework",
 				"${BUILT_PRODUCTS_DIR}/FLKAutoLayout/FLKAutoLayout.framework",
-				"${BUILT_PRODUCTS_DIR}/HockeySDK-Source/HockeySDK_Source.framework",
```

Comment:
@ashfurrow what creates this diff? Building the project?

Reply:
This is modified during `pod install`. Since we removed the HockeySDK dependency, CocoaPods has updated our project to remove references to it.